sonarts-core/tests/runRule.ts

```diff
@@ -51,6 +51,24 @@ export interface RuleRunResult {
  */
 export default function runRule(Rule: any, testFileName: string, ...ruleArguments: any[]): RuleRunResult {
   const lintFileName = getLintFileName(testFileName);
+  return checkRule(Rule, lintFileName, ruleArguments);
+}
+
+/**
+ * Run rule againts a specific lint file from a test directory
+ * Use from the test: runRule(Rule, "myTestDir", "myTestFile"), ommitting '.lint.ts' suffix on test file
+ */
+export function runRuleWithLintFile(
+  Rule: any,
+  testDirName: string,
```

Comment:
@m-g-sonar it's dir here, and ruleName in deeper call. Let's keep one abstraction